src/Microsoft.AspNetCore.Mvc.RazorPages/Internal/PageLoggerExtensions.cs

```diff
@@ -38,6 +40,16 @@ static PageLoggerExtensions()
                 102,
                 "Executed handler method {HandlerName}, returned result {ActionResult}.");
 
+            _implicitHandlerMethodExecuting = LoggerMessage.Define<ModelValidationState>(
+                LogLevel.Information,
+                103,
+                "Executing an implicit handler method - ModelState is {ValidationState}");
+
+            _implicitHandlerMethodExecuted = LoggerMessage.Define<string>(
+                LogLevel.Debug,
```

Comment:
`LogLevel.Information` Begin and end events should have the same log level. In addition, all-`Information` matches the corresponding "explicit handler" events in `MvcCoreLoggerExtensions`.

Reply:
Also changing the log level of `_handlerMethodExecuted` right above to `Information`.